csi.proto

```diff
@@ -1767,6 +1767,14 @@ message VolumeGroupSnapshot {
   // Timestamp of when the volume group snapshot was taken.
   // This field is REQUIRED.
   .google.protobuf.Timestamp creation_time = 3;
+
+  // Indicates if all individual snapshots in the group snapshot
+  // are ready to use as a `volume_content_source` in a
+  // `CreateVolumeRequest`. The default value is false.
+  // CO MUST wait until all snapshots are ready to use before
+  // setting this field to true.
```

Comment:
If any snapshot in the list of snapshots in this message have a ready_to_use set to false, the SP MUST set this field to false.
If all of the snapshots in the list snapshots in this message have ready_to_use set to true, the SP SHOULD set this field to true.

Reply:
Done